prediction_market_agent/tools/mech/utils.py

```diff
@@ -62,12 +44,15 @@ class MechTool(str, Enum):
 
 
 def mech_request(question: str, mech_tool: MechTool) -> MechResponse:
-    private_key = MechAPIKeys().bet_from_private_key.get_secret_value()
+    private_key = APIKeys().bet_from_private_key.get_secret_value()
```

Comment:
Is this function still used somewhere? My understanding is that the `general_agent` is not relying on the Mech for doing predictions, correct?

Reply:
Yes, it's still used by `PredictProbabilityForQuestionMech` (see [here](https://github.com/gnosis/prediction-market-agent/blob/evan/remove-mech/prediction_market_agent/agents/microchain_agent/market_functions.py#L146-L150)), which itself is not used by the general agent atm, but I didn't want to delete the code until https://github.com/gnosis/prediction-market-agent/issues/327 was done/closed as wont-fix